app/src/main/java/ch/sdp/vibester/activity/LyricsBelongGameActivity.kt

```diff
@@ -80,36 +66,53 @@ class LyricsBelongGameActivity : GameActivity() {
         }
     }
 
+    private fun setCheckButtonListener(ctx: Context){
+        findViewById<Button>(R.id.lyricMatchButton).setOnClickListener {
+            getAndCheckLyrics(ctx, songName, artistName, speechInput, gameManager)
+        }
+    }
+
+    private fun setNextButtonListener(ctx: Context, gameManager: GameManager) {
+        findViewById<Button>(R.id.nextSongButton).setOnClickListener {
+            startRound(ctx, gameManager)
+        }
+    }
+
```

Comment:
Not urgent, but maybe refactor these two functions into one "set listeners of buttons" function.